game/scripts/npc/heroes/dragon_knight.txt

```diff
@@ -9,5 +9,6 @@
     //-------------------------------------------------------------------------------------------------------------
     "Ability5"                                            "dragon_knight_elder_dragon_form" // replaces generic_hidden
     "Ability6"                                            "dragon_knight_elder_dragon_form_oaa" // replaces dragon_knight_elder_dragon_form
+    "Ability16"                                           "special_bonus_unique_dragon_knight_6" // replaces "special_bonus_unique_dragon_knight"
```

Comment:
Additional splash percent as lvl 25 talent? It seems really bad. I suggest buffing Dragon Blood talent or replacing it with Dragon Blood aura talent if you want to go nuts.

Reply:
Dragon Blood aura talent seemed disgusting, it doesn't get disabled if DK gets broken. Additional splash is not an amazing talent, but DK has 0 damage talents. All of his talents are either for his stun and/or worthless. Theres another talent for frost breath ms and attack slow but it seemed hard to make work. Theres no way to make a 25 talent for dragons blood that doesn't make DK an unkillable behemoth. You already didn't like me trying to give him 8 more regen.

Reply:
ok here's my proposal. Swap fire breath dmg reduction from 10 to 25, and bring splash damage to level 15, health to level 10.

level 10 talent: 200 health or 15 damage
level 15: splash or stun
level 20: same as before
level 25: aoe stun or 75% dmg reduction (100% total, super dispellable but at least it's worth the 25 talent, and accomplishes similar goals to the dragon blood talent, better for tanking bosses, being tanky, also helps your team out more though)

Reply:
I liked 2x regen and armor but ye the straight numbers values kinda suck. We have to choose what direction we want DK. Tanky with some damage, or Damage with some tanky.